src/main/scala/uk/gov/hmrc/ct/ct600e/v2/E24e1.scala

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2016 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.ct.ct600e.v2
+
+import uk.gov.hmrc.ct.box._
+import uk.gov.hmrc.ct.ct600e.v2.retriever.CT600EBoxRetriever
+
+case class E24e1(value: Option[Int]) extends CtBoxIdentifier("Loans and non-trade debtors (Held at the end of the period)") with CtOptionalInteger with Input with ValidatableBox[CT600EBoxRetriever] {
```

Comment:
Can we make these E24eA and E24eB instead? This is the standard we used for the V3 E170 boxes.